x/ecocredit/core/types_project.go

```diff
@@ -0,0 +1,33 @@
+package core
```

Comment:
No changes here. Moved from `core/genesis.go` (and what is now `genesis/genesis.go`).